src/sentry/static/sentry/app/views/performance/compare/spanGroup.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+
+import {TreeDepthType} from 'app/components/events/interfaces/spans/types';
+
+import {DiffSpanType, SpanGeneratedBoundsType} from './utils';
+import SpanBar from './spanBar';
+
+type Props = {
+  span: Readonly<DiffSpanType>;
+  renderedSpanChildren: Array<JSX.Element>;
+  treeDepth: number;
+  continuingTreeDepths: Array<TreeDepthType>;
+  spanNumber: number;
+  isLast: boolean;
+  isRoot: boolean;
+  numOfSpanChildren: number;
+  generateBounds: (span: DiffSpanType) => SpanGeneratedBoundsType;
+};
+
+type State = {
+  showSpanTree: boolean;
+};
+
+class SpanGroup extends React.Component<Props, State> {
+  state: State = {
+    showSpanTree: true,
+  };
+
+  toggleSpanTree = () => {
+    this.setState(state => ({
+      showSpanTree: !state.showSpanTree,
+    }));
+  };
+
+  renderSpanChildren = () => {
```

Comment:
This doesn't need to be bound, it isn't used as event handler/prop.

Reply:
should be addressed in https://github.com/getsentry/sentry/pull/18610/commits/969a783ebb2dc947317e040c22e70622b35fa2b7